tests/test_packageAlerts.py

```diff
@@ -0,0 +1,389 @@
+        self.diaObjects["pm_parallax_Cov"] = None
+        self.diaObjects["uLcPeriodic"] = None
+        self.diaObjects["gLcPeriodic"] = None
+        self.diaObjects["rLcPeriodic"] = None
+        self.diaObjects["iLcPeriodic"] = None
+        self.diaObjects["zLcPeriodic"] = None
+        self.diaObjects["yLcPeriodic"] = None
+        self.diaObjects["uLcNonPeriodic"] = None
+        self.diaObjects["gLcNonPeriodic"] = None
+        self.diaObjects["rLcNonPeriodic"] = None
+        self.diaObjects["iLcNonPeriodic"] = None
+        self.diaObjects["zLcNonPeriodic"] = None
+        self.diaObjects["yLcNonPeriodic"] = None
+
+        self.diaSources = diaSourceHistory.loc[
+            [(0, "g", 8), (1, "g", 9)], :]
+        self.diaSourceHistory = diaSourceHistory.drop(labels=[(0, "g", 8),
+                                                              (1, "g", 9)])
+
+    def testMakeCutoutBytes(self):
+        """Test round tripping an exposure/cutout to bytes and back.
+        """
+        packageAlerts = PackageAlertsTask()
+
+        sphPoint = self.exposure.getWcs().pixelToSky(self.center)
+        cutout = self.exposure.getCutout(sphPoint, packageAlerts.cutoutBBox)
+
+        cutoutBytes = packageAlerts.makeCutoutBytes(cutout)
+        tempMemFile = afwFits.MemFileManager(len(cutoutBytes))
+        tempMemFile.setData(cutoutBytes, len(cutoutBytes))
+        cutoutFromBytes = afwImage.ExposureF(tempMemFile)
+        self.assertTrue(cutout, cutoutFromBytes)
+
+    def testMakeJsonAlert(self):
+        """Test stripping data from the various data products and into a
+        dictionary "alert".
+        """
+        packageAlerts = PackageAlertsTask()
+        alertId = 1234
+
+        for srcIdx, diaSource in self.diaSources.iterrows():
+            sphPoint = geom.SpherePoint(diaSource["ra"],
+                                        diaSource["decl"],
+                                        geom.degrees)
+            cutout = self.exposure.getCutout(sphPoint,
+                                             packageAlerts.cutoutBBox)
+            cutputBytes = packageAlerts.makeCutoutBytes(cutout)
+            objSources = self.diaSourceHistory.loc[srcIdx[0]]
+            alert = packageAlerts.makeJsonAlert(
+                alertId,
+                diaSource,
+                self.diaObjects.loc[srcIdx[0]],
+                objSources,
+                cutout,
+                None)
+            self.assertEqual(len(alert), 9)
+
+            self.assertEqual(alert["alertId"], alertId)
+            self.assertEqual(alert["diaSource"], diaSource.to_dict())
+            self.assertEqual(alert["cutoutDifference"]["stampData"],
+                             cutputBytes)
+            self.assertEqual(alert["cutoutTemplate"],
+                             None)
+
+    def testRun(self):
+        """Test the run method of package alerts.
+        """
+        packConfig = PackageAlertsConfig()
+        tempdir = tempfile.mkdtemp(prefix='alerts')
```

Comment:
I don't think this tempdir ever gets cleaned up. Might be better to put this in `setUp` so you can `tearDown` it.